packages/model-viewer/src/three-components/Reticle.ts

```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
  */
 
-import {Camera, Math as ThreeMath, Matrix4, Mesh, MeshBasicMaterial, Object3D, Raycaster, RingGeometry, Vector3,} from 'three';
+import {Camera, MathUtils as ThreeMath, Matrix4, Mesh, MeshBasicMaterial, Object3D, Raycaster, RingGeometry, Vector3,} from 'three';
```

Comment:
Minor: You can probably just import `MathUtils` and rename all the `ThreeMath`s.